internal/kafka/command_mirror_truncate_and_restore.go

```diff
@@ -0,0 +1,63 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	kafkarestv3 "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *mirrorCommand) newTruncateAndRestoreCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "truncate-and-restore <local-topic-1> [local-topic-2] ... [local-topic-N]",
+		Short:             "Truncate and restore local topics.",
```

Comment:
This needs to be the same as what's in https://github.com/confluentinc/cli/pull/2859/files#diff-f497b5a85ada34224fa1d523b2f4acd9afd66c27982cd813fba1ff25dd6379d3R1. 

I think what's in the other file is quite long and redundant, so instead, let's just use this for everything:

```
Truncate the local topic and restore mirroring to fetch from the remote topic.
```